org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -942,20 +939,12 @@ public Preferences setJavaCompletionFavoriteMembers(List<String> javaCompletionF
 	}
 
 	public Preferences putSha256(List<String> sha256Allowed, List<String> sha256Disallowed) {
-		List<String> oldAllowed = this.sha256Allowed;
-		List<String> oldDisallowed = this.sha256Disallowed;
 		WrapperValidator.clear();
 		this.sha256Allowed = sha256Allowed;
 		if (sha256Disallowed != null) {
 			WrapperValidator.disallow(sha256Disallowed);
```

Comment:
this needs to move to the Gradle preference listener now

Reply:
Fixed.